tests/BasicTests.elm

```diff
@@ -273,6 +280,64 @@ twoChildFormDecoded =
             }
 
 
+keyedNodeWithText : Html.Html msg
+keyedNodeWithText =
+    Keyed.ul [] [ ("0", Html.text "Hello!") ]
+
+
+keyedNodeWithTextDecoded : NodeType
+keyedNodeWithTextDecoded =
+    NodeEntry
+        { tag = "ul"
+        , children = [ TextTag { text = "Hello!" } ]
+        , descendantsCount = 1
+        , facts = emptyFacts
+        }
+
+
+keyedNodeWithChildren : Int -> Html.Html msg
+keyedNodeWithChildren childrenCount =
+    let
+        liWithText val =
+            ( toString val, Html.li [] [ Html.text (toString val) ] )
+    in
+        Keyed.ul [] <|
+            List.map liWithText [1..childrenCount]
+
+
+keyedNodeWithChildrenDecoded : Int -> NodeType
+keyedNodeWithChildrenDecoded childrenCount =
```

Comment:
`keyedNodeWithChildrenDecoder 2` does not make clear sense of what the function is doing. Please give a comment explaining